packages/frontend/src/app.tsx

```diff
@@ -1,6 +1,44 @@
+import type { ReactNode } from "react";
```

Comment:
Just as a reminder, this file should not be changed

Reply:
I've undone these changes now.